File: A1/A1-029/A1-029.cpp
#include<iostream>
using namespace std;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);
    // Input
    string text;
    cin >> text;
    // Count vowel letters
    int cnt = 0;
    for(int i = 0; i < text.size(); i++)
    {
        if(text[i] == 'a' || text[i] == 'e' || text[i] == 'i' || text[i] == 'o' || text[i] == 'u')
        {
            cnt++;
        }
    }
    // Output
    cout << cnt;
    return 0;
}

File: A1/A1-029/A1-029.py
# --------------------------------------------------
# File Name : A1-029.py
# Problem   : Count Vowels
# Author    : Worralop Srichainont
# Date      : 2026-03-13
# --------------------------------------------------

# Vowel constants
VOWELS = ("a", "e", "i", "o", "u")

# Input and converts all to lowercase
text = input().strip().lower()

# Count vowels
vowel_count = 0
for char in text:
    if char in VOWELS:
        vowel_count += 1

# Output
print(vowel_count)
